modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+
+Run a system check to ensure that your system meets Redpanda's requirements.
+
+Example:
+
+```bash
+sudo rpk redpanda check
+```
+
+Output:
+```bash
+System check results
+CONDITION                                          REQUIRED      CURRENT          SEVERITY  PASSED
+Ballast file present                               true          true             Warning   true
+Clock Source                                       tsc           tsc              Warning   true
+Config file valid                                  true          true             Fatal     true
+Connections listen backlog size                    >= 4096       4096             Warning   true
+Data directory filesystem type                     xfs           xfs              Warning   true
+Data directory is writable                         true          true             Fatal     true
+Data partition free space [GB]                     >= 10         1755.29          Warning   true
+Dir '/var/lib/redpanda/data' IRQs affinity set     true          true             Warning   true
+Dir '/var/lib/redpanda/data' IRQs affinity static  true          true             Warning   true
+Dir '/var/lib/redpanda/data' nomerges tuned        true          true             Warning   true
+Dir '/var/lib/redpanda/data' scheduler tuned       true          true             Warning   true
+Free memory per CPU [MB]                           2048 per CPU  7659             Warning   true
+Fstrim systemd service and timer active            true          true             Warning   true
+I/O config file present                            true          true             Warning   true
+Kernel Version                                     3.19          5.15.0-1056-aws  Warning   true
+Max AIO Events                                     >= 1048576    1048576          Warning   true
+Max syn backlog size                               >= 4096       4096             Warning   true
+NIC IRQs affinity static                           true          true             Warning   true
+NTP Synced                                         true          true             Warning   true
+RFS Table entries                                  >= 32768      32768            Warning   true
+Swap enabled                                       true          true             Warning   true
+Swappiness                                         1             1                Warning   true
+Transparent huge pages active                      true          true             Warning   true
+```
+
+[[redpanda-lmr]]
+=== Redpanda's latest version
+
+Check that Redpanda is running on the xref:https://github.com/redpanda-data/redpanda/releases[latest maintenance release^] on every node.
+
+Example:
+```bash
+/usr/bin/redpanda --version
+```
+
+Output:
+```bash
+v23.3.9 - f0a24fa4b60d55bd075c291e213807b5dfa74c0a
+```
+
+[[cpu-memory]]
+=== Correct CPUs and Memory configured
+
+Ensure that the correct number of CPUs and memory are being used by Redpanda.
```

Comment:
Hmmm...memory is not typically referred to as a "correct number". How about:

Ensure that you have the correct number of CPUs and sufficient memory.

Reply:
How does the user know what this"correct number" and sufficient allocation for memory are? This is a check, but where can they go to see what these should be in the first place?

Thinking every check here should end with a link to the appropriate section that tells users how to set up their config so they will be able to pass these checks. Thoughts @tmgstevens @hcoyote ?

Reply:
> How does the user know what this"correct number" and sufficient allocation for memory are? This is a check, but where can they go to see what these should be in the first place?

For this one, it's the user who must define that. For example, if they're on a system that has 8CPUs and they're fully dedicated to redpanda, then the correct number is 8. If it's half Redpanda half another application, it will be 4/4. Only the user can define what's the correct number.

As for memory, I believe you can allocate memory upfront, and again only the user would know how much he allocated. 

@hcoyote this here might be a good idea to say where the user can define such parameters and what are our current Least Amount Required.